src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -18,6 +18,12 @@ A NTFS feed is not sufficient to generate a valid Netex Profil France feed:
 - `ParticipantRef` : The ParticipantRef code must be provided,
 - `StopProviderCode` : The code used to identify the provider in the stop_id generation.
 
+## Id of objects
+
+According to the Netex Specification, object identifiers should have a specific formating.
+The formating for stops is describe in the [arrets.xml](#arretsxml) part of this specification.
+For the other objects, the following format is used: **FR:[object type]:[object id]:**
```

Comment:
The `object_id` might create problems if it contains `:`. We need to replace all `:` (with `_` I suppose, like for stops' identifiers?).